### ISSUE TITLE
Handle cloned clients

### ISSUE THREAD

**Issue by johnpmitsch** [RedHatSatellite/satellite-clone#171]
In a cloned machine, clients still exist in the database, but are not altered by the satellite (i.e attaching subscriptions). This is my experience in my very limited testing with this situation, so we should make sure of this before anything else.

We need to have a way to handle clients on a clone, should we be changing anything?

**Comment by johnpmitsch** [comment]
We need to test this further

**Comment by sthirugn** [comment]
I did more testing on clients, and this is what I found.  With the following results, I think hosts are safe in the cloned satellite atleast until the hostname of the original satellite is changed to something new.

- Deleting the registered content host from cloned satellite does not affect the registered host at all.
- Unregistering the content host also did not affect the registered client.
- Tried deleting a provisioned libvirt vm and a vmware vm and I got the following error. I think this is trying to delete tftp entry from the original satellite and it fails.
2017-04-20 16:45:44 [app] [W] TFTP Settings for testhost11.test.com task failed with the following error: ERF12-7299 [ProxyAPI::ProxyException]: Unable to delete TFTP boot entry for 52:54:00:a0:66:a1 ([RestClient::ResourceNotFound]: 404 Resource Not Found) for Capsule https://satellite.example.com:9090/tftp

**Now clients on external capsules need to be validated.**
